web/src/app/administration/components/access-items-management/access-items-management.component.ts

```diff
@@ -40,7 +39,7 @@ export class AccessItemsManagementComponent implements OnInit {
     ['workbasket-key', 'Workbasket Key']
   ]);
   sortModel: Sorting = new Sorting('access-id', Direction.DESC);
-  isGroup: boolean = false;
+  accessItems: any[];
```

Comment:
Is it possible to determine a type?

Reply:
and 'isGroup' is used in html line 121.

Reply:
If I set the accessItems to type accessItemWorkbasket, what it's supposed to be, an error occurs and thus it doesn't recognizes the belonging parameters anymore (I have no idea why). since the name of the variable is already quite meaningful, I would maybe just stick with the type any?

Reply:
Should every object from the class AccessItemWorkbasket have the attribute accessName? If this is the case, we should add the attribute accessName to the class AccessItemWorkbasket.